app/schemas/FOOD.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "foodId": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "nutritionalData": {
+      "type": "object",
+      "properties": {
+        "calories": {
+          "type": "integer"
+        },
+        "protein": {
+          "type": "string"
+        },
+        "carbs": {
+          "type": "string"
+        },
+        "fats": {
+          "type": "string"
+        },
+        "fiber": {
+          "type": "string"
+        }
+      },
+      "required": ["calories", "protein", "carbs", "fats", "fiber"]
+    },
+    "menuId": {
+      "type": "string"
+    }
+  },
+  "required": ["foodId", "name", "description", "nutritionalData", "menuId"]
+}
```

Comment:
I think we're going to want a `purchase_at_url` that stores the URL where the user can buy the food from. 
1. Use NutriDine to filter and find the food meeting your macros
2. Redirect off NutriDine to doordash, restaurant website...etc to make the purchase